src/post-message.ts

```diff
@@ -258,11 +258,15 @@ const unsubscribe = (listener: MessageListener) => {
     messageListeners = messageListeners.filter((f) => f !== listener);
 };
 
+const isDisabledFromIframe = () =>
+    frameElement?.hasAttribute('data-disable-webview-bridge') ?? false;
```

Comment:
check that frameElement (or window) exists, otherwise this isn't SSR friendly

Reply:
Perhaps it would be better to disable by default and explicitly enable it with a data attribute

Reply:
yes, I prefer to disable it by default inside iframes

Reply:
Sure, I did it this way to ensure zero impact on current consumers.
I'll change it 👍